### DISCUSSION TITLE
Is Plausible Cloud down?

### DISCUSSION THREAD

**Discussion by sptv2reg** [plausible/analytics#2668] [Anything Else]
Hi

We're getting Error 502 on requests to Plausible. The main site is also unavailable for us.

Anyone else seeing this?

![image](https://user-images.githubusercontent.com/103186488/218439545-08c52475-c1ce-4928-9476-0045e3cacc03.png)

**Comment by sptv2reg** [comment]
I could have checked the statuspage which was updated (not really used to that from Google Analytics :-))

I wonder if you have any plans for securing (extra) against data loss in the future? It really is critical for media sites (which I know is a good match for Plausible). I imagine that data could be collected in a log and batch processed later if the backend went down. Is there something like that in the long term planning?

**Comment by metmarkosaric** [comment]
sorry about that @sptv2reg! we had some issues few minutes ago but things are back up and running all fine now. we've been growing very fast so we're putting a lot of effort into our infrastructure to make everything run as smooth and as fast as possible at all times. we're working on some type of queue functionality at the same time that will allow us to keep tracking events even when there are some issues. thanks for your patience with this!
